Server/src/dfs/DFS.java

```diff
@@ -1,3 +1,4 @@
+package dfs;
 
```

Comment:
shouldn't be deleting entire file and readding it, pulling from master should help